include/deal.II/meshworker/simple.h

```diff
@@ -107,11 +107,12 @@ namespace MeshWorker
       template<class DOFINFO>
       void assemble(const DOFINFO &info1,
                     const DOFINFO &info2);
-    private:
+    protected:
```

Comment:
Since I need to rewrite the assembly routine in the derived class `SystemSimple` to take inhomogeneous constraints into account, I need access to the matrix, threshold, and residuals data of the parent classes. As Martin asked for, I can also reuse the constraint object from either `MatrixSimple` or `ResidualSimple`. Then this needs to get protected too...